packages/SwingSet/src/controller.js

```diff
@@ -164,21 +164,25 @@ export async function makeSwingsetController(
       return;
     }
 
-    // Add a timestamp to the slog input object.
+    // Create an object with the property order that loadgen expects.
+    const timedObj = { time: obj.time, type: obj.type, ...obj };
```

Comment:
Why do we care about plucking the `obj.time` since we override it right after? (If I had to guess, it'd be to satisfy the typing system which would believe it'd be a certain value?)

Reply:
That was leftovers from a prior attempt.  I fixed it now.